in_toto/rulelib.py

```diff
@@ -285,3 +288,7 @@ def pack_allow_rule(pattern):
 def pack_disallow_rule(pattern):
   """Shortcut for 'pack_rule' to pack a DISALLOW rule. """
   return pack_rule("DISALLOW", pattern)
+
+def pack_require_rule(pattern):
+  """Shortcut for 'pack_rule' to pack a REQUIRE rule. """
+  return pack_rule("REQUIRE", pattern)
```

Comment:
Do you mind changing `pattern` to `filename`?

Reply:
Should we maybe add your *"REQUIRE is a weird animal"*-[note](https://github.com/in-toto/in-toto-golang/blob/3578e27ff208bd4cc0135a34ab6213532cd55ca8/in_toto/verifylib.go#L287-L288) to the docstrings of `unpack_rule` (return value), and `pack_rule` (arguments), mentioning that `pattern` may  not always be a glob pattern?